src/components/Header/TopNavbar/TopNavConst.tsx

```diff
@@ -0,0 +1,74 @@
+import { useNavigate } from "react-router-dom";
+import { FindIconUrl } from "../../../utils/Constant";
+import { NavbarDirection } from "../../types";
+
+// eslint-disable-next-line no-warning-comments
+// eslint-disable-next-line react-hooks/rules-of-hooks
+const navigate = useNavigate();
+
+export interface IReviewRating {
+  average_rating: number;
+  total_review_count: number;
+}
+export interface IDataTopNav {
+  brand: string;
+  discount_percent_of_first_variant: string | number | null;
+  discounted_price_of_first_variant: string | number | null;
+  id: number;
+  image: string;
+  is_favorite: boolean;
```

Comment:
Sometime we are using underscore case sometime camelCase. Please do not use underscore case